_posts/2022-04-19-debt-reduction-in-the-product-roadmap.md

```diff
@@ -0,0 +1,78 @@
+---
+layout: post
+title: "How to include refactoring into Product development roadmap? Reducing technical debt inspired by real options identification"
+author: [olga.dudzik]
+tags: [tech]
+---
+
+## Nowadays, technical debt can be considered a bread and butter of most IT-powered enterprises around the world.
```

Comment:
Pierwszy akapit będzie używany jako zajawka posta więc najlepiej żeby było jedno-dwa zdania przed pierwszym śródtytułem. Jeśli chcesz mieć od razu śródtytuł, to w nałówku plik umd trzeba dodać pole `excerpt: "..."` i tam podać tekst, który będzie pokazywany jako zajawka